modin/pandas/indexing.py

```diff
@@ -296,6 +338,14 @@ def __getitem__(self, row_lookup, col_lookup, ndim):
             The global row index to retrieve data from.
         col_lookup : slice(None), range or np.ndarray
             The global col index to retrieve data from.
+        row_multiindex_full_lookup : bool
+            See _multiindex_possibly_contains_key.__doc__.
+        col_multiindex_full_lookup : bool
+            See _multiindex_possibly_contains_key.__doc__.
```

Comment:
```suggestion
        row_multiindex_full_lookup : bool
            Whether `row_lookup` is potentially an all-levels lookup.
        col_multiindex_full_lookup : bool
            Whether `col_lookup` is potentially an all-levels lookup.
```
?

Reply:
I think the _multiindex_possibly_contains_key is more useful here.

Reply:
`_multiindex_possibly_contains_key` seems ok to me, too.